ppl/archive/import.go

```diff
@@ -69,22 +65,22 @@ type Writer struct {
 // NewWriter creates a zbuf.Writer compliant writer for writing data to an
 // archive.
 func NewWriter(ctx context.Context, ark *Archive) *Writer {
-	g, ctx := errgroup.WithContext(ctx)
 	ctx, cancel := context.WithCancel(ctx)
+	g, ctx := errgroup.WithContext(ctx)
 	return &Writer{
-		ark:          ark,
-		cancel:       cancel,
-		ctx:          ctx,
-		errgroup:     g,
-		flushTimeout: DefaultImportFlushTimeout,
-		writers:      make(map[tsDir]*tsDirWriter),
+		ark:      ark,
+		cancel:   cancel,
+		ctx:      ctx,
+		errgroup: g,
+		writers:  make(map[tsDir]*tsDirWriter),
 	}
 }
 
-// SetFlushTimeout sets the interval at which the Writer will wait to flush
-// stale records to disk. This must be called before writing any records.
-func (w *Writer) SetFlushTimeout(timeout time.Duration) {
-	w.flushTimeout = timeout
+// SetStaleDuration sets the StaleThreshold for the writer which is the
+// duration since a tsdir has last since received new data will  flushed to
```

Comment:
```suggestion
// delay after the last write to a tsdir until it is flushed to
```

Reply:
Thanks man, words on tough for me today.